### PR TITLE
Testing To Do list: part 2

### PR DESCRIPTION
**Pull request by silviatofana**
General requirements
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
JavaScript requirements
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
Use jest framework for testing.
Create a test file(s) ([..].test.js) for a file(s) containing the following functions:
a function for editing the task description.
a function for updating an item's 'completed' status.
the "Clear all completed" function.
Use the mock storage object you have created in Part 1 of this project to mock the localStorage updates.
Mock the HTML elements to test DOM manipulation functions.
Make sure you group your tests using the describe() method.
Optional requirements
Check the test coverage. You should aim for covering 100% of your code.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,10 +24,13 @@
...
-## Collaborators
+👤 Grace Muthui
 
+- GitHub: [@Grayce Muthui](https://github.com/Graycemuthui)
+- Twitter: [@Grayce_Muthui](https://twitter.com/Grayce_Muthui)
+- LinkedIn: [@Grayce Muthui](http://www.linkedin.com/in/grayce-muthui-a17294226)
 
+## Collaborators
 
 ## 🤝 Contributing
```
Comment (Deepakdanger) [inline README.md:35]: - [ ] I would recommend including the steps on how to test the application in the README file. [Hint: npm test]

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @silviatofana & @Graycemuthui, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- All tests are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (silviatofana) [reply README.md]: ok, thanks for the helpful advice I will include in the README file how to test the application. [Hint: npm test]

**Review by t-yanick — APPROVED** [review]
Hi @silviatofana, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
woow thanks

**Pull request merged**